src/extensions/default/UploadFiles/UploadFilesDialog.js

```diff
@@ -4,6 +4,8 @@
 define(function (require, exports, module) {
     "use strict";
 
+    var _uploadDialog = null;
```

Comment:
They (Adobe) used it to separate so-called local variables and things that might get used externally.  It's all over the Brackets code, so I think it's useful to do it.